angular/projects/admin/src/app/admin/document/document-form/document-form.component.ts

```diff
@@ -124,24 +125,29 @@ export class DocumentFormComponent implements OnInit, OnDestroy, OnChanges {
   }
 
   async saveEntry() {
-    const formData = this.documentForm.value;
-    console.log(`[DocumentEditComponent:saveEntry] ${JSON.stringify(formData)}`);
-
-
-    const updates = {
-      id: this.document.id,
-      title: formData.title,
-      documentType: this.document.documentType,
-      status: formData.status,
-      data: this.dataForm.value,
-      url: formData.url,
-    } as Document;
-    await this.documentService.update(updates);
-
+      const formData = this.documentForm.value;
+      const formIsValid = this.authorIsValid !== undefined ? this.documentForm.valid && this.authorIsValid : this.documentForm.valid ;
```

Comment:
Form is valid if all mandatory fields are valid. The only fields that we know that every document has are the following

- `status`
- `title`
 - `url`
 - `published` (optional)
 - `updated`
 - `created`

Most of those fields "can't" be invalid since they are automatic values.

So what you need to do is to loop through all fields in `data` and make sure that all of them are "valid" ... not only `author`. It must be a generic check.